### PR TITLE
Update R package to 0.16.0

### PR DESCRIPTION
**Pull request by rpkyle**
This PR proposes to update `dashHtmlComponents` to 0.16.0. This package enforces a dependency on `dash` rather than `dashR`.

### CODE REVIEW

File: R/htmlA.R [hunk 1]
```diff
@@ -15,4 +15,5 @@ htmlA <- function(children=NULL, id=NULL, n_clicks=NULL, n_clicks_timestamp=NULL
     component$props <- filter_null(component$props)
 
     structure(component, class = c('dash_component', 'list'))
-}
\ No newline at end of file
+}
+
```
Comment (alexcjohnson) [inline R/htmlA.R]: are these extra trailing lines intentional? It's nice to have one trailing newline, but 2 is unusual.

**Comment by rpkyle** [comment]
> Love the typing info and extra line breaks in the long arg list! The comment about double-trailing-newlines would be nice but not blocking. 💃

😌 I have to admit things have come a long way from those initial package versions.

I'll fix up the double-trailing-newline in my Dash PR and recommit, should be corrected momentarily!

File: R/htmlA.R (hunk 1 above)
Comment (rpkyle) [reply R/htmlA.R]: fixed in ec6fd2b

**Comment by alexcjohnson** [comment]
Sweet 💃

**Pull request merged**